### ISSUE TITLE
Helm Trident Operator - "tridentNodePluginTolerations" will not be used for Windows Daemonset

### ISSUE THREAD

**Issue by burkhat** [NetApp/trident#828]
**Describe the bug**
We're using the latest helm version 23.4.0 to rollout the trident operator with windows support.
In Openshift it is necessary to set tolerations in the Windows DaemonSet that Pods gets started.

We tried following config in the values.yaml:
```
tridentNodePluginTolerations:
  - operator: Exists
    effect: NoExecute
  - operator: Exists
    effect: NoSchedule
```
But the toleration will only be set in the Linux DaemonSet and not in the Windows DaemonSet.
Workaround: Deploy the DaemonSet with the tolerations manually

**Environment**
Provide accurate information about the environment to help us reproduce the issue.

- Trident version: 23.04
- Container runtime: cri-o
- Kubernetes version: 1.25
- Kubernetes orchestrator: Openshift 4.12
- Kubernetes enabled feature gates: -
- OS: RHEL 8, Windows Server 2022
- NetApp backend types: ONTAP-NAS
- Other:

**To Reproduce**
See Bug desscription

**Expected behavior**
Tolerations will be set in the Windows DaemonSet, too

**Additional context**

**Comment by travisghansen** [comment]
I also need this one. It may make sense to have a different set of tolerations per OS though..

**Comment by travisghansen** [comment]
Can we get an update on this?

**Comment by travisghansen** [comment]
Can we get this looked at before the 23.10 release?

**Comment by mravi-na** [comment]
This issue is fixed with https://github.com/NetApp/trident/commit/5efa9949b0863566c17014a41723088b14e32b99 and is included in 23.10
